gps2val/gpsd_feeder.py

```diff
@@ -59,7 +59,7 @@ def setData(self, data):
             if v is not None:
                 self.client.setValue(k,str(v))
                
-class GPSD_Client():
+class GPSD_Client_Instance():
```

Comment:
Having "instance" in a class name is confusing as an object is an instance of a class already.
Also, it seems to me this class could be a subclass of `threading.Thread` and that the `loop` method could be replaced with an override of https://docs.python.org/3.8/library/threading.html?highlight=thread#threading.Thread.run.
We could then rename the class `GPSDClientThread` (without underscores which are forbidden in class names).